include/open62541/server.h

```diff
@@ -85,6 +85,9 @@ UA_Server_run_iterate(UA_Server *server, UA_Boolean waitInternal);
 UA_StatusCode UA_EXPORT
 UA_Server_run_shutdown(UA_Server *server);
 
+void *
+UA_Server_getNsCtx(UA_Server *server);
```

Comment:
to use the nodestore interface declared in nodestore.h (for example _UA_Nodestore_newNode(void* nsCtx, UA_NodeClass nodeClass)_ ) the nodestoreContext is needed. Therefore I've added this call in server.h. Do you think there is a better way to do this?

Reply:
UA_Nodestore* now available from serverConfig, so I removed this hack.